modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/ExponentialBackoff.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.persistence.pagemem;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Implements exponential backoff logic. Contains a counter and on each {@link #nextDuration()} increments it.
+ * May be reset using {@link #reset()}.
+ */
+class ExponentialBackoff {
```

Comment:
Lets rename the class to `ExponentialBackoffCalculator` as the whole purpose of the class is simply calculating throttling intervals lenghts.

Reply:
It's not just a calculator, it also holds state (backoff counter), so `Calculator` seems non-optimal. So it represents the concrete instance of an exponential backoff process, and just `ExponentialBackoff` seems an adequate name.